axelor-budget/src/main/resources/views/SaleOrderLine.xml

```diff
@@ -75,42 +81,17 @@
   </action-group>
 
   <action-group name="action-budget-sale-order-line-group-onload">
-    <action name="action-sale-order-line-supplychain-group-onload"/>
+    <action name="action-supplychain-group-sale-saleorderline-onload"/>
     <action name="action-method-sale-order-line-budget-check-budget-key-roles"
       if="__config__.app.isApp('budget')"/>
   </action-group>
 
-  <action-group name="action-group-sale-saleorderline-product-onchange"
-    id="action-budget-group-sale-saleorderline-product-onchange">
-    <action name="action-sale-order-line-method-get-product-information"/>
-    <action name="action-sale-order-line-method-compute"/>
-    <action name="action-sale-order-line-attrs-hide-bill-of-material"/>
-    <action name="action-sale-order-line-get-production-information"/>
-    <action name="action-sale-order-line-method-supplier-partner-default"
-      if="__config__.app.isApp('supplychain')"/>
-    <action name="action-sale-order-line-method-check-stock"
-      if="__config__.app.isApp('supplychain') &amp;&amp; __config__.app.getApp('supplychain')?.getCheckSaleStocks()"/>
-    <action name="action-sale-order-line-method-check-qty"
-      if="__config__.app.getApp('sale')?.manageMultipleSaleQuantity"/>
-    <action if="__config__.app.isApp('supplychain')"
-      name="action-sale-order-line-attrs-show-delivery-panel"/>
-    <action name="action-method-sale-order-line-available-and-allocated-stock"/>
-    <action name="action-sale-order-attrs-hide-ati"/>
-    <action name="action-record-sale-order-line-future-qty"
-      if="__config__.app.isApp('supplychain')"/>
-    <action name="action-sale-order-line-record-complementary-products-to-handle"/>
-    <action name="action-sale-order-line-method-fill-max-discount"/>
-    <action name="action-sale-order-line-method-translate-product-description-and-name"/>
+  <action-group name="action-budget-group-sale-sale-orderline-product-onchange">
```

Comment:
Please rename to `budget-action-sale-order-line-group-product-onchange` to match naming conventions